src/Domain/CoordinateMaps/CoordinateMap.hpp

```diff
@@ -336,6 +336,22 @@ class CoordinateMap
     return lhs.maps_ == rhs.maps_;
   }
 
+  template <typename NewMap, size_t... Is>
```

Comment:
why do the impls not go in the tpp?

Reply:
:man_facepalming: yea, it should be in the `tpp`... Thanks :)